src/dev/flang/be/c/C.java

```diff
@@ -605,6 +605,40 @@ private void createCode(CFile cf)
                                                  r.ret()))));
     var ordered = _types.inOrder();
 
+    // start of fileio related intrinsics
+    // fuzion.std.fileio.read
+    // currently logs console messages instead of proper returns in case of errors
+    cf.print
+      (CStmnt.lineComment("helper to read bytes from a file"));
+      cf.print
+        ("void "+CNames.FILEIO_READ._name+"""
+        (char* path, void* content, size_t size)
+        {
+          FILE *f = fopen(path, \"r\");
+          if (f == NULL)
+          {
+            fprintf(stderr, \"*** failed to open the file %s \\n\", path);
+            exit(0);
+          }
+          else 
+          {
+            size_t reading = fread(content, 1, size, f);
+            if (reading != size)
+            {
+              if (ferror(f)!=0)
+              {
+                fprintf(stderr, \"*** failed to read the file %s \\n\", path);
+              }
+              else if (feof(f)!=0)
+              {
+                fprintf(stdout, \"reached end of the file %s \\n\", path);
```

Comment:
Please add a comment describing why this code is here, e.g.,  `/* NYI: remove debug output */`